disgordrouter/context.go

```diff
@@ -43,13 +43,14 @@ func (c *Context) Get(key string) interface{} {
 
 // Reply replies to the sender with the given message
 func (c *Context) Reply(args ...interface{}) (*disgord.Message, error) {
-	return c.Ses.SendMsgString(c.Msg.ChannelID, fmt.Sprint(args...))
+	Message, Error := c.Ses.SendMsg(c.Msg.ChannelID, fmt.Sprint(args...))
+	return Message, Error
```

Comment:
Simply `return c.Ses.SendMsg(c.Msg.ChannelID, fmt.Sprint(args...))` here.
It will return both values of the function

Reply:
I thought it was supposed to also, for some reason though it kept saying it wasn't returning enough arguments that way when I was checking it. I will try it again. I am new to Go and still learning, so maybe it was something else that it was complaining about and I misunderstood.